models/timeTracker.model.js

```diff
@@ -0,0 +1,34 @@
+const mongoose = require('mongoose');
+
+mongoose.Promise = global.Promise;
+
+const timeTrackerSchema = mongoose.Schema({
+    hours: { type: Number },                // limit to whole numbers or .5 ?
```

Comment:
it's better to calculate hours in the code using the start time and end time. Let's remove this field.

Reply:
++